openamp/overview.rst

```diff
@@ -61,32 +69,43 @@ Runtime Control
 
 With the domains defined, **runtime control** of the asymmetric slave applications can be started. The master will load the images as required. In this example the RTOS image could be loaded at power on to perform say environmental instrument monitoring and the bare metal image on demand to perform some specific high intensity calculations, but stopped on completion for power savings.
 
+.. _ipc-work-label:
+
 Inter Processor Communications
 ==============================
 
 .. raw:: html
     :file: ../images/fundamentals/ipc.svg
 
-`Inter Processor Communications <https://en.wikipedia.org/wiki/Inter-process_communication>`_ is performed through shared memory and is between master and slave. In this star topology example the slaves can not communicate with each other. If that were required a chain topology would allow one remote to contain a slave and a master in which case they could communicate.
+`Inter Processor Communications <https://en.wikipedia.org/wiki/Inter-process_communication>`_ is performed through shared memory and is between master and slave. In this star topology example the slaves cannot communicate with each other. If that were required a chain topology would allow one remote to contain a slave and a master in which case they could communicate.
+
```

Comment:
master/slave  to replace in the whole file

Reply:
should be done and will search for any existing as further changes